widgets/post-carousel/tpl/base.php

```diff
@@ -34,6 +34,7 @@
 		     data-page="1"
 		     data-fetching="false"
 		     data-dir="<?php echo is_rtl() ? 'rtl' : 'ltr'; ?>"
+		     data-center-items="<?php echo esc_attr( $center_items ) ?>"
```

Comment:
Should this be `esc_attr( $center_items );` with a closing `;`?

Reply:
`;` isn't required in this situation and I left it out to match the previous line. Regardless, I'll change this (and the previous line).